berkeley-function-call-leaderboard/data/possible_answer/BFCL_v3_live_multiple.json

```diff
@@ -950,7 +950,7 @@
 {"id": "live_multiple_949-199-0", "ground_truth": [{"play_song": {"query": ["David Guetta"], "shuffle": ["", false], "repeat": ["", false], "volume": ["", 70]}}]}
 {"id": "live_multiple_950-199-1", "ground_truth": [{"set_alarm": {"alarm_time": ["23:00"], "purpose": ["", "General reminder"]}}]}
 {"id": "live_multiple_951-200-0", "ground_truth": [{"version_api.VersionApi.get_version": {}}]}
-{"id": "live_multiple_952-201-0", "ground_truth": [{"uber.eat.order": {"restaurant": ["McDonald's"], "items": [["fries"]], "quantities": [[5]], "notes": [""], "delivery_option": ["", "standard"]}}]}
+{"id": "live_multiple_952-201-0", "ground_truth": [{"uber.eat.order": {"restaurant": ["McDonald's"], "items": [["fries"]], "quantities": [[5]], "notes": ["", ""], "delivery_option": ["", "standard"]}}]}
```

Comment:
There are some instances of this ["", ""].

Why make this change @JasonHuang1103